### PR TITLE
12 factor app

### PR DESCRIPTION
**Pull request by akanagar**

### CODE REVIEW

File: documentation/12-factor-app-devon4node.asciidoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+
+= 12-factor-app with devon4j
+
+This document mainly focuses on discussing how can you create 12 factor app with devon4j. To know more about this 12 factors you can refer https://12factor.net/[here] . Twelve factor is mainly focus on creating cloud native applications. These are the guidelines on what factors you need to consider in different stages of application lifecycle.
```
Comment (sjimenez77) [inline documentation/12-factor-app-devon4node.asciidoc:8]: devon4node instead devon4j

File: documentation/12-factor-app-devon4node.asciidoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+
+= 12-factor-app with devon4j
```
Comment (sjimenez77) [inline documentation/12-factor-app-devon4node.asciidoc:6]: devon4node instead devon4j

File: documentation/12-factor-app-devon4node.asciidoc [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+|Factor |Action
+
+|Codebase
+|*One codebase tracked in revision control, many deploys* +
+
+One codebase should be created with each process or service.
+Twelve factor app says do not share code between application. +
+If you must share code, build library and manage that through dependencies. +
+
+To create and manage library in spring boot refer https://docs.nestjs.com/cli/libraries[here]  +
```
Comment (sjimenez77) [inline documentation/12-factor-app-devon4node.asciidoc:22]: Spring Boot? devon4node is based on NestJS

File: documentation/12-factor-app-devon4node.asciidoc [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+Dockerize Nestjs app refer  https://dev.to/abbasogaji/how-to-dockerize-your-nestjs-app-for-production-2lmf[here].
+
+Dockerize Node app refer 
+https://blog.appsignal.com/2021/10/19/how-to-dockerize-an-existing-nodejs-application.html[here]
+
+|Config
+|*Store config in the environment*
+
+Here, configuration means anything which can vary between different deployment such as credential to external sources, backing services etc. To do such configuration in your java application refer guide https://docs.nestjs.com/techniques/configuration[here]
```
Comment (sjimenez77) [inline documentation/12-factor-app-devon4node.asciidoc:42]: java application?

**Review by sjimenez77 — CHANGES_REQUESTED** [review]
I am not reviewing anymore since it seems that this document is a copy paste from another one from devon4j. Please, check the document to update references from java, spring boot or any java related topic. It is very important to make a first review of your work first before creating a PR .

**Comment by akanagar** [comment]
will create new one

**Pull request closed without merging**